js/keyboard/keyboard.js

```diff
@@ -0,0 +1,147 @@
+import {
+	LANGUAGE,
+} from '../constants.js';
+import buttons from './buttonData.js';
+import Button from './button.js';
+
+export default class Keyboard {
+	constructor(outputEl, language) {
+		this.language = language || LANGUAGE.EN;
+		this.outputEl = outputEl;
+		this.buttons = [];
```

Comment:
`this.buttons` should not be in this class at all. All the pieces of code here (in this class) should be the methods in the class `Button`. All the 'eventListeners' started from `this.buttons` should be added in the class 'Button'. It helps to reduce class dependency on each other.